solutions/devicesimulation-nohub/single-vm/setup.sh

```diff
@@ -80,14 +80,14 @@ SETUP_URL="${REPOSITORY}/setup/"
 
 install_docker_ce() {
     apt-get update -o Acquire::CompressionTypes::Order::=gz \
-        && apt-get upgrade -y \
+        && sudo DEBIAN_FRONTEND=noninteractive apt-get upgrade -y \
```

Comment:
I don't think the sudo is needed here. Also not sure if it needs to be in the same line. I would probably put in a separate line.

To mitigate I'll approve but separating the command and removing sudo I think makes sense.

Reply:
Fixed. I tried to move the DEBIAN_FRONTEND to its own line, but that didn't work (it appears that it has to be in line with the apt-get command). You're right about the sudo thought, that's not needed.